dao/src/integ-test/java/uk/gov/moj/sdt/dao/PurgeNativeCallFunctionTest.java

```diff
@@ -0,0 +1,177 @@
+import uk.gov.moj.sdt.dao.api.IBulkSubmissionDao;
+import uk.gov.moj.sdt.dao.api.IIndividualRequestDao;
+import uk.gov.moj.sdt.dao.config.DaoTestConfig;
+import uk.gov.moj.sdt.domain.BulkSubmission;
+import uk.gov.moj.sdt.domain.ErrorLog;
+import uk.gov.moj.sdt.domain.GlobalParameter;
+import uk.gov.moj.sdt.domain.IndividualRequest;
+import uk.gov.moj.sdt.domain.PurgeJobAudit;
+import uk.gov.moj.sdt.domain.PurgeJobAuditMessage;
+import uk.gov.moj.sdt.domain.ServiceRequest;
+import uk.gov.moj.sdt.test.utils.AbstractIntegrationTest;
+import uk.gov.moj.sdt.test.utils.TestConfig;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test {@link PurgeNativeCallFunction} query methods.
+ *
+ * @author Mark Dathorne
+ */
+@ActiveProfiles("integ")
+@SpringBootTest(classes = { TestConfig.class, DaoTestConfig.class})
+@Sql(scripts = {"classpath:db/migration/V0002__purge.sql",
+        "classpath:uk/gov/moj/sdt/dao/sql/PurgeDaoTest.sql"})
+@Transactional
+class PurgeNativeCallFunctionTest extends AbstractIntegrationTest {
+    /**
+     * Logger object.
+     */
+    private static final Logger LOGGER = LoggerFactory.getLogger(PurgeNativeCallFunctionTest.class);
+
+    private PurgeNativeCallFunction purgeNativeCallFunction;
+
+    private PurgeJobAuditDao purgeJobAuditDao;
+    private PurgeJobAuditMessageDao purgeJobAuditMessageDao;
+    private ErrorLogDao errorLogDao;
+    private GlobalParametersDao globalParametersDao;
+    private ServiceRequestDao serviceRequestDao;
+    private IBulkSubmissionDao bulkSubmissionDao;
+    private IIndividualRequestDao individualRequestDao;
+
+    @BeforeEach
+    public void setup() {
+        purgeNativeCallFunction = this.applicationContext.getBean(PurgeNativeCallFunction.class);
+
+        purgeJobAuditDao = (PurgeJobAuditDao) this.applicationContext.getBean("PurgeJobAuditDao");
+        purgeJobAuditMessageDao = (PurgeJobAuditMessageDao) this.applicationContext.getBean("PurgeJobAuditMessageDao");
+        errorLogDao = (ErrorLogDao) this.applicationContext.getBean("ErrorLogDao");
+        globalParametersDao = (GlobalParametersDao) this.applicationContext.getBean("GlobalParametersDao");
+        serviceRequestDao = (ServiceRequestDao) this.applicationContext.getBean("ServiceRequestDao");
+        bulkSubmissionDao = (IBulkSubmissionDao) this.applicationContext.getBean("BulkSubmissionDao");
+        individualRequestDao = (IIndividualRequestDao) this.applicationContext.getBean("IndividualRequestDao");
+    }
+
+    /**
+     * Tests {@link GenericDao} fetch.
+     */
+    @Test
+    void runPurge() {
+        LOGGER.info("TEST STARTING: run the Purge");
+
+        checkTestDataBeforeRunningPurge();
+
+        purgeNativeCallFunction.executePurgeStoredProc(1000);
+
+        checkTestDataAfterRunningPurge();
+
+        LOGGER.info("TEST COMPLETED: the Purge has been run");
+    }
+
+    private void  checkTestDataBeforeRunningPurge() {
```

Comment:
There's a double space between "void" and the method name.  Same for checkTestDataAfterRunningPurge().